Add note on tag requirements (#448)
hashicorp/terraform-docs-common: website/docs/cloud-docs/workspaces/creating.mdx

For CLI and API-driven workspaces, the workspace overview page appears. For version control workspaces, the **Configure Terraform variables** page appears.

### Workspace Tags

> Once you have created a workspace you can create or assign tags to it. Here are some things to note about tags:

Reviewer: Can you please apply my above suggestion to this list as well?
```suggestion
After you create a workspace, you can create or assign tags to it. Tags are subject to the following rules:
```
